src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -47,6 +47,15 @@
                 "JobPollingFrequency": "00:00:10",
                 "MaximumNumberOfResourcesPerQuery": 100,
                 "NumberOfPagesPerCommit": 10
+            },
+            "Reindexing": {
+                "Enabled": true,
```

Comment:
Should this be disabled by default? Since `AcquireReindexJobsAsync` will throw a NotImplementedException we will end up throwing exceptions while the worker is running